android/app/src/main/kotlin/com/zulip/flutter/ZulipPlugin.kt

```diff
@@ -87,7 +87,7 @@ private class AndroidNotificationHost(val context: Context)
                         // FlutterLocalNotificationsPlugin, which handles receiving the Intent.
                         // TODO take care of receiving the notification-opened Intent ourselves
```

Comment:
Also, I see that this special name "payload" remains after removing `flutter_local_notifications`. Is that because we're happy/neutral to keep it around, or is it a sign that there's still helpful cleanup of `flutter_local_notifications` to do?

Reply:
Updated with main commit (`notif: Handle opening of notification via URI intents`) removing dangling references for `payload`.